.github/workflows/main_ci.yml

```diff
@@ -4,9 +4,11 @@ on:
   push:
     branches:
       - pydarshan-devel
+      - "**"
   pull_request:
     branches:
       - pydarshan-devel
+      - "**"
```

Comment:
That seems reasonable--note that this is a change to the CI config itself, so you'll only see it take effect once it is merged.

A logical sequence of events might have been:
- a simple PR to do just this change to activate the CI on this feature branch
- subsequent PRs against this branch can then be made with CI visible/running